--- a/src/download.sh	2010-02-11 16:22:48.000000000 +0100
+++ b/src/download.sh	2010-03-24 13:20:47.000000000 +0100
@@ -29,7 +29,7 @@
 set -e
 
 VERSION="0.9.1"
-MODULES="rapidshare megaupload 2shared badongo mediafire 4shared zshare depositfiles storage_to uploaded_to uploading netload_in usershare sendspace x7_to hotfile divshare freakshare dl_free_fr loadfiles"
+MODULES="rapidshare depositfiles storage_to x7_to"
 OPTIONS="
 HELP,h,help,,Show help info
 GETVERSION,v,version,,Return plowdown version
